<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medCare</class>
 <widget class="QMainWindow" name="medCare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(:/images/images/fon.jpg);
}
#widget{
background-color:rgb(203,239,227);
border-radius:20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(214,196,182);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page">
          <widget class="QPushButton" name="settings_cancelButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>141</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
	padding:10px 5px;
	border-radius:5px;</string>
           </property>
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="settings_nameLine">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>140</y>
             <width>221</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
padding-left: 7px;</string>
           </property>
          </widget>
          <widget class="QLabel" name="settings_nameLabel">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Имя&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="settings_surnameLine">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>220</y>
             <width>221</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
padding-left: 7px;</string>
           </property>
          </widget>
          <widget class="QLabel" name="settings_surnameLabel">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>220</y>
             <width>81</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Логин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="settings_townLine">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>310</y>
             <width>221</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
padding-left: 7px;</string>
           </property>
          </widget>
          <widget class="QLabel" name="settings_townLabel">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>310</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Город&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="settings_saveButton">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>420</y>
             <width>201</width>
             <height>71</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
	padding:10px 5px;
	border-radius:5px;</string>
           </property>
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
          <widget class="QLabel" name="settings_nameErrorLabel">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>195</y>
             <width>201</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="settings_surnameErrorLabel">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>270</y>
             <width>211</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="settings_townErrorLabel">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>370</y>
             <width>231</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qtimages/photo.qrc"/>
 </resources>
 <connections/>
</ui>
